dev/com.ibm.websphere.appserver.features/visibility/private/explore-1.0/com.ibm.websphere.appserver.adminCenter.tool.explore-1.0.feature

```diff
@@ -1,5 +1,6 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=com.ibm.websphere.appserver.adminCenter.tool.explore-1.0
+WLP-DisableAllFeatures-OnConflict: false
 visibility=private
```

Comment:
Should not need the visibility setting when including a `.feature` file in the `com.ibm.websphere.appserver.features/visibility/private` folder.  The build of the `.feature` file should add this automatically based on the visibility folder it is located.